src/actions/navigation/index.js

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) 2016 Mattermost, Inc. All Rights Reserved.
+// See License.txt for license information.
+
+import {NavigationTypes} from 'constants';
+import Routes from 'navigation/routes';
+
+export function goBack() {
+    return async (dispatch, getState) => {
+        dispatch({
+            type: NavigationTypes.NAVIGATION_POP
+        }, getState);
+    };
+}
+
+export function goToLogin() {
+    return async (dispatch, getState) => {
+        dispatch({
+            type: NavigationTypes.NAVIGATION_PUSH,
+            route: Routes.Login
+        }, getState);
+    };
+}
+
+export function goToSelectTeam() {
+    return async (dispatch, getState) => {
+        dispatch({
+            type: NavigationTypes.NAVIGATION_PUSH,
+            route: Routes.SelectTeam
+        }, getState);
+    };
+}
+
+export function goToChannel() {
```

Comment:
If i understand correctly we should pass here the channel to which we want to switch, right?